numpy/core/tests/test_scalarmath.py

```diff
@@ -184,7 +184,6 @@ def test_modular_power(self):
         a = 5
         b = 4
         c = 10
-        expected = pow(a, b, c)
```

Comment:
Would be inclined to leave this as documentation. Feel free to stick whatever the comment is that flake8 needs to ignore this line.